go.mod

```diff
@@ -36,6 +35,8 @@ require (
 	github.com/lightningnetwork/lightning-onion v0.0.0-20180605012408-ac4d9da8f1d6
 	github.com/ltcsuite/ltcd v0.0.0-20190101042124-f37f8bf35796
 	github.com/miekg/dns v0.0.0-20171125082028-79bfde677fa8
+	github.com/ory/go-acc v0.0.0-20181118080137-ddc355013f90 // indirect
```

Comment:
Is this right, that it is a module requirement of `lnd` now?

Reply:
this was the way go.mod and go.sum were left as a result of the coverage run. i don't think it's strictly necessary to include them, and am happy to remove them. for instance, i don't think they'd show up when just doing a regular `make && make install`

Reply:
Yeah I don't think this is needed, it's only a result of the script being run in the same directory.